schemas/errors/generator.py

```diff
@@ -0,0 +1,98 @@
+import json
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List
+
+
+@dataclass
+class ErrorDefinition:
+    code: str
+    http_status_code: int
+    description: str
+
+
+class ErrorGenerator(ABC):
+    @abstractmethod
+    def generate(self, errors: List[ErrorDefinition]) -> str:
+        """Generate error code implementation from error definitions"""
+        pass
+
+    @abstractmethod
+    def get_output_directory(self) -> str:
+        """Get the output directory"""
+        pass
+
+    @abstractmethod
+    def get_output_file_name(self) -> str:
+        """Get the output file name"""
+        pass
+
+
+class PythonErrorGenerator(ErrorGenerator):
+    def generate(self, errors: List[ErrorDefinition]) -> str:
+        code = [
+            "from enum import Enum",
+            "from dataclasses import dataclass",
+            "",
+            "@dataclass",
+            "class ErrorDetails:",
+            "    code: str",
+            "    http_status_code: int",
+            "",
+            "class ErrorCode(Enum):",
+        ]
+
+        for error in errors:
+            code.append(f"    # {error.description}")
+            code.append(
+                f'    {error.code} = ErrorDetails(code="{error.code}", http_status_code={error.http_status_code})'
+            )
+            code.append("")
+
+        return "\n".join(code)
+
+    def get_output_directory(self) -> str:
+        return "../uma-python-sdk/uma/generated"
+
+    def get_output_file_name(self) -> str:
+        return "errors.py"
+
+
+def load_errors(json_path: str) -> List[ErrorDefinition]:
+    """Load and parse error definitions from JSON file"""
+    with open(json_path, "r") as f:
+        data = json.load(f)
+        return [
+            ErrorDefinition(
+                code=error["code"],
+                http_status_code=error["httpStatusCode"],
+                description=error["description"],
+            )
+            for error in data.get("errors", [])
+        ]
+
+
+def generate_error_codes():
+    """Generate error codes from input JSON to output directory"""
+    errors = load_errors("schemas/errors/errors.json")
+    generators = [PythonErrorGenerator()]
+
+    for generator in generators:
+        output_path = Path(generator.get_output_directory())
+        output_path.mkdir(parents=True, exist_ok=True)
```

Comment:
I'm not sure whether we want to do think given my question above, but let's talk through it.